### PR TITLE
Add ISO section key

### PR DESCRIPTION
**Pull request by irth**
support for the § key on ISO keyboards :)

### CODE REVIEW

**Comment by envomer** [comment]
@soffes Can we get this merged? Thanks ahead ^^

File: Sources/HotKey/Key.swift [hunk 1]
```diff
@@ -260,6 +261,7 @@ public enum Key {
 		case "rightarrow", "→": self = .rightArrow
 		case "downarrow", "↓": self = .downArrow
 		case "uparrow", "↑": self = .upArrow
+        case "section", "§": self = .section
```
Comment (soffes) [inline Sources/HotKey/Key.swift:264]: Mind fixing the indentation on 264, 384, and 504 to match the others around it?

**Review by soffes — COMMENTED** [review]
Thanks for this! I don't have an ISO keyboard so this never crossed my mind!

Happy to merge after a minor style nit :)

**Comment by soffes** [comment]
Oh realizing this is an old PR. Will go ahead and merge

**Comment by soffes** [comment]
Released in version 0.2.1

**Comment by envomer** [comment]
Thank you very much 😍

**Pull request merged**